src/components/recommendations-overlay/index.js

```diff
@@ -1,3 +1,11 @@
-import RecommendationsOverlay from './recommendations-overlay';
-
-export default RecommendationsOverlay;
+export default async function lazyComponent(player) {
```

Comment:
This will make things easier for global search and for readability 
```suggestion
export default async function lazyRecommendationsOverlay(player) {
```

Reply:
👍 
only that it's a component, not a plugin :)